test/hls/hls_parser_integration.js

```diff
@@ -22,11 +25,24 @@ describe('HlsParser', () => {
   /** @type {!shaka.test.Waiter} */
   let waiter;
 
+  function checkClearKeySupport() {
+    // Some versions of Tizen doesn't support CBCS, so omit it for now.
+    // See: https://github.com/shaka-project/shaka-player/issues/1419
+    if (shaka.util.Platform.isTizen()) {
+      return false;
+    }
+    return support['org.w3.clearkey'];
+  }
+
   beforeAll(async () => {
     video = shaka.test.UiUtils.createVideoElement();
     document.body.appendChild(video);
     compiledShaka =
         await shaka.test.Loader.loadShaka(getClientArg('uncompiled'));
+    const responses = await Promise.all([
+      shaka.media.DrmEngine.probeSupport(),
+    ]);
+    support = responses[0];
```

Comment:
This doesn't make sense to me.  Why Promise.all() on a single Promise?  Shouldn't it be `support = await shaka.media.DrmEngine.probeSupport()`?

Reply:
Changed!